cvat/requirements/base.txt

```diff
@@ -51,5 +51,5 @@ av==6.2.0
 patool==1.12
 boto3
 opencv-contrib-python
-onepanel-sdk==0.14.0b
+onepanel-sdk==0.15.0b2
```

Comment:
should this be 0.15.0 or maybe upcoming 0.16.0?

Reply:
Since 15.0 is out, I could use that? Since 0.16 is not out yet, I can't build the image.

Reply:
> Since 15.0 is out, I could use that?

Yes, I think that makes the most sense. We're going to remove `is_valid_token` in 0.16.0.